lib/graphql/language/parser.y

```diff
@@ -165,7 +165,6 @@ rule
 
   arguments_opt:
       /* none */                    { return EMPTY_ARRAY }
-    | LPAREN RPAREN                 { return EMPTY_ARRAY }
```

Comment:
I believe this changes removes matching `field()` if I got the understanding of racc correctly.